spec/controllers/v2/link_sets_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+RSpec.describe V2::LinkSetsController do
+  let(:content_id) { SecureRandom.uuid }
+  let(:link_type) { "topic" }
```

Comment:
These `let`'s don't add much, because they're only used once per test. Maybe clearer to inline them?